src/components/pages/Groups/IdentitiesPanelForm/IdentitiesPanelForm.tsx

```diff
@@ -36,6 +39,25 @@ const IdentitiesPanelForm = ({
   removeIdentities,
   setRemoveIdentities,
 }: Props) => {
+  const { data } = useGetIdentities();
```

Comment:
This is just a partial implementation of selecting identities from the API to unblock this and other work.